nik4.py

```diff
@@ -621,6 +623,18 @@ def run(options):
     parser.add_argument('output', help='Resulting image file')
     options = parser.parse_args()
 
+    if options.tiles:
+        if re.search(r'^\d+$', options.tiles):
+            options.tiles_x = int(options.tiles)
+            options.tiles_y = options.tiles_x
+        else:
+            match = re.search(r'^(\d+)x(\d+)$', options.tiles)
+            if (match):
```

Comment:
Better without brackets:
```suggestion
            if match:
```